### PR TITLE
Added indexes to WorldBankTariffPipeline

### PR DESCRIPTION
**Pull request by marcuspp**
### Description of change
This change adds indexes to several columns on the world_bank_tariffs table and removes the prf_rate column. 

I tested this locally and the indexes are created without issue the only concern I had is when the tables get swapped and renamed. The indexes will keep their original names (like sequences currently do) but I don't think this will be an issue as the temp tables are named with a timestamp so next run will be able to create indexes without issue. 

### Checklist

* [ ] Have tests been added to cover any changes?
* [ ] Has the [CHANGELOG](https://github.com/uktrade/data-flow/blob/master/CHANGELOG.md) been updated?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: dataflow/dags/dataset_pipelines.py [hunk 1]
```diff
@@ -687,16 +687,16 @@ class WorldBankTariffPipeline(_DatasetPipeline):
 
     table_name = 'world_bank_tariffs'
     source_url = f'{config.DATA_STORE_SERVICE_BASE_URL}/api/v1/get-world-bank-tariffs/?orientation=records'
+    allow_null_columns = True
```
Comment (allait) [inline dataflow/dags/dataset_pipelines.py]: Why is this needed?

Comment (marcuspp) [reply dataflow/dags/dataset_pipelines.py]: Initially we might only load a subset of the dataset so that it can be verified. So there is a chance one of the rate types could be missing. Once we are loading all data this can be removed.
